src/Input/Select/Select.noTypeChecking.test.tsx

```diff
@@ -46,7 +46,7 @@ describe('<Select /> can still work when receives non-string option children or
     const selectInput = queryByRole('combobox');
     userEvent.type(selectInput, '1');
     const filteredOptions = queryAllByTestId('option');
-    expect(filteredOptions.length).toEqual(1);
+    expect(filteredOptions.length).toEqual(3);
```

Comment:
@farhanabi how is it this changed?

Reply:
Basically, I removed the filtering that is happening on non-string options because otherwise it will not show anything when the children are not string, even though we passed the options. We could manipulate this by just changing the options passed directly.